lib/forms/overlay.vue

```diff
@@ -132,7 +132,7 @@
         </ui-tabs>
         <div v-else class="input-container-wrapper" :style="{ 'max-height': `calc(100vh - ${formTop} - 56px)`}">
           <div class="input-container">
-            <field v-for="(field, fieldIndex) in sections[0].fields" :key="fieldIndex" :name="field" :data="fields[field]" :schema="schema[field]" :initialFocus="initialFocus"></field>
+            <field v-for="(field, fieldIndex) in sections[0].fields" :key="fieldIndex" :name="field" :data="fields[field]" :schema="schema[field] || getFieldSchema(field)" :initialFocus="initialFocus"></field>
```

Comment:
I weakly prefer move the `schema[field]` here to the `getFieldSchema()` method just because I don't like in-template programming.

Reply:
if we call `getFieldSchema()`, logic will run on every render. This way the `schema[field]` logic is actually only run when `schema` or `field` change